### ISSUE TITLE
Number of epochs before seeing a decrease in the loss

### ISSUE THREAD

**Issue by Kev1MSL** [TencentARC/InstantMesh#113]
Hi, I am currently training the instant-mesh model but after 3 epochs I am not seeing convergence and the loss does not seem to decrease. I am using the same config as you set (DDP strategy) and also I added grad_accumulation. I am not sure if I am doing something wrong or if I should be more patient and wait for more epochs. 

Here some plots if it helps:
![image](https://github.com/TencentARC/InstantMesh/assets/82373568/82397095-323d-4249-840e-a91a09d1c838)
![image](https://github.com/TencentARC/InstantMesh/assets/82373568/16a05aa3-dd68-495d-966e-cfb32ced0584)

Thanks for your help!

**Comment by bluestyle97** [comment]
You need to train instant-nerf first. The mesh-based rendering can only provide gradients at near-surface area, making the network hard to converge. Our mesh model is finetuned from the nerf model.

**Comment by Kev1MSL** [comment]
Alright makes sense thank you! I have tried the training of instant-nerf, however I am starting with cubes as first images and I was wondering if the default code starts with pretrained weights for the LRM since I am starting with cubes?

**Comment by Sri-vatsa** [comment]
@Kev1MSL According to the paper, the training is initialized using openlrm weights. @bluestyle97 How many training steps have you run the training of instant-nerf for using your training data?

**Comment by Mrguanglei** [comment]
@Kev1MSL Hello, please take a look at your dataset structure and training configuration file？ This will help you better

**Comment by Mrguanglei** [comment]
@Sri-vatsa Hello, could you take a look at the structure of your dataset and the configuration file used to train nerf? I have encountered a problem in this regard, which I cannot solve, and I need your help

**Comment by HaFred** [comment]
> Alright makes sense thank you! I have tried the training of instant-nerf, however I am starting with cubes as first object rendered and I was wondering if the default code starts with pretrained weights for the LRM since I am starting with cubes?

Hi based on your graph, it seems like your training set contains only 3.5k instances, is that true? The paper says 270k instances, so do you know how to curate the dataset correctly for training instantnerf? Thanks a lot!

**Comment by Jinyiyi3** [comment]
> You need to train instant-nerf first. The mesh-based rendering can only provide gradients at near-surface area, making the network hard to converge. Our mesh model is finetuned from the nerf model.

Does fine-tuning the InstantMesh require first fine-tuning the NeRF model, even if the my goal is solely to improve the performance of the Mesh Model?
